application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/wolframalpha/WolframAlphaCommand.java

```diff
@@ -0,0 +1,374 @@
+        List<Pod> pods = new ArrayList<>(result.getPods());
+        List<MessageAction> messages = new ArrayList<>();
+        logger.info("There are {} pods", pods.size());
+        int imageNo = 0;
+        OUTER: for (int i = 0; i < pods.size();) {
+            Pod pod = pods.get(i);
+            List<SubPod> subPods = new ArrayList<>(pod.getSubPods());
+            logger.info("pod number {}", ++i);
+            logger.info("There are {} sub pods within this pod", subPods.size());
+            for (int j = 0; j < subPods.size();) {
+
+                SubPod subPod = subPods.get(j);
+                logger.info("sub pod number {}", ++j);
+                WolfImage image = subPod.getImage();
+                try {
+                    String name = image.getTitle();
+                    String source = image.getSource();
+                    String extension = ".png";
+                    String header = pod.getTitle();
+                    boolean jIs1 = j == 1;
+                    int width =
+                            (jIs1 ? Math.max(getWidth(header), image.getWidth()) : image.getWidth())
+                                    + 10;
+                    int height = image.getHeight();
+                    if (j == 1)
+                        height += TEXT_HEIGHT;
+                    BufferedImage readImage =
+                            new BufferedImage(width, height, BufferedImage.TYPE_4BYTE_ABGR);
+                    Graphics graphics = readImage.getGraphics();
+                    if (jIs1) {
+                        graphics.setFont(WOLFRAM_ALPHA_FONT);
+                        graphics.setColor(Color.WHITE);
+                        graphics.setColor(WOLFRAM_ALPHA_TEXT_COLOR);
+                        graphics.drawString(header, 10, 15);
+                    }
+                    var srcImg = ImageIO.read(new URL(source));
+                    graphics.drawImage(srcImg, 10, jIs1 ? 20 : 0, null);
+
+                    ImageIO.write(readImage, "png", Path
+                        .of("C:\\Users\\Abc\\IdeaProjects\\TJ-Bot-baseRepo\\application\\src\\main\\java\\org\\togetherjava\\tjbot\\commands\\mathcommands\\wolframalpha\\img%d.png"
+                            .formatted(++imageNo))
+                        .toFile());
+                    ImageIO.write(srcImg, "png", Path
+                        .of("C:\\Users\\Abc\\IdeaProjects\\TJ-Bot-baseRepo\\application\\src\\main\\java\\org\\togetherjava\\tjbot\\commands\\mathcommands\\wolframalpha\\ogImg%d.png"
+                            .formatted(imageNo))
+                        .toFile());
+
+                    if (name.isEmpty()) {
+                        name = header;
+                    }
+                    name += extension;
+                    maxWidth = Math.max(maxWidth, width);
+                    // FIXME get the attachments <= 10 or return a collection
+                    if (filesAttached == 10) {
+                        break OUTER;
```

Comment:
```suggestion
                        break OUTER;
```
If possible, separate the loop with a method where you return something instead of breaking using a label.